lib/resultsCapturer.js

```diff
@@ -58,15 +58,13 @@ module.exports = function () {
   });
 
   this.registerHandler('StepResult', function (event, callback) {
```

Comment:
But they all seem like events. Cucumber's registerHandler function takes this function and reassigns to the 'handler' variable and passes it to a function called setHandlerForEvent. 

Naming this one to stepResult reads okay but for the Before* handlers it reads a little weird. Maybe they become more specific, such as beforeFeatureEvent, stepResultEvent, etc...? I'm ok with leaving them as 'event' too.